canonical-web-and-design/charmhub.io
# Refactor the card pattern to utilise Vanilla more

## Done
Removed the custom card styling and replaced it with the native Vanilla card with the addition of two utilities which have to propose to Vanilla.

@solazio I wanted to show how I would do it so a PR seemed the best way. Please, pick it apart. 

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8045
- See that the cards match [the design](https://app.zeplin.io/project/5d3f1850b938ee0dc24c60c4/screen/5e3984c0bdbcb10d1946c78f)
